docs/index.md

```diff
@@ -3,8 +3,8 @@ layout: page
 title: CCACommander Ultra Promax Xtra 9000PLUS
 ---
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
-[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S1-CS2103T-F11-1/tp/actions)
```

Comment:
I think the badge svg should point towards our own repo

Reply:
Ok, will change the link!